recipes/pytables/meta.yaml

```diff
@@ -0,0 +1,79 @@
+{% set version="3.2.2" %}
+
+package:
+  name: pytables
+  version: {{ version }}
+
+source:
+  fn: PyTables-v.{{ version }}
+  url: https://github.com/PyTables/PyTables/archive/v.{{ version }}.tar.gz
+  sha256: 2626e874caa6b3fcf2bfc28b9dd6a40a3f859c72e19ce0764a60a6d77e350008
+  patches:
+    # remove march=native, fix library paths
+    - setup.patch          # [win]
+    # fix a syntax error
+    - blosc_macro.patch
+    # teach it to look for dlls in Library\bin on win
+    - tables_init.patch    # [win]
+    # remove march=native for more widely compatible builds
+    - march.patch          # [unix]
+
+build:
+  number: 0
+  entry_points:
+    - pt2to3 = tables.scripts.pt2to3:main
+    - ptdump = tables.scripts.ptdump:main
+    - ptrepack = tables.scripts.ptrepack:main
+    - pttree = tables.scripts.pttree:main
+
+requirements:
+  build:
+    - python
+    - cython
+    - zlib      1.2*
+    - lzo                      # [linux]
+    - bzip2
+    - numexpr
+    - hdf5      1.8.15*
+    - setuptools
+    # building blosc is done internally.
+    # this is an old version, though.
+    # would be better to link against new version externally.
+    # See https://github.com/PyTables/PyTables/issues/451
+    # - blosc                  # [win]
+    - numpy     x.x
+  run:
+    - python
+    - argparse                 # [py26]
+    - numpy     x.x
+    - numexpr
+    - hdf5      1.8.15*
+    - zlib      1.2*
+    - bzip2                    # [win]
+    # building blosc is done internally.
+    # this is an old version, though.
+    # would be better to link against new version externally.
+    # See https://github.com/PyTables/PyTables/issues/451
+    # - blosc                  # [win]
+
+test:
+  requires:
+    - unittest2                # [py26]
+  commands:
+    - pt2to3 -h
+    - ptdump -h
+    - ptrepack -h
+    - pttree -h
+
+about:
+  home: http://www.pytables.org
+  license: BSD 3-clause
+  summary: brings together Python, HDF5 and NumPy to easily handle large amounts of data
+
+extra:
+  recipe-maintainers:
+    - avalentino
+    - FrancescAlted
+    - jakirkham
```

Comment:
If it's all the same, I think I will pass on this one. While I use `h5py` quite a bit, I don't really use `pytables` and I may already be stretching myself thin.